config/routes.rb

```diff
@@ -30,6 +30,8 @@
       post "activate", to: "staff#activate"
       post "update_activation", to: "staff#update_activation"
     end
+
+    get "task/cancel", to: "tasks#cancel"
```

Comment:
I believe we can keep the routes together by doing
```
resources :tasks, only: [:new, :create, :edit, :update, :destroy] do
  collection do
    get "cancel", to: "tasks#cancel"
  end
end
```

Reply:
I completely forgot about collection!